packages/calcite-components/src/components/action/action.tsx

```diff
@@ -102,6 +117,11 @@ export class Action extends LitElement implements InteractiveComponent {
   /** Use this property to override individual strings used by the component. */
   @property() messageOverrides?: typeof this.messages._overrides;
 
+  /**
+   * When `true` and the `type` is `"toggle"`, the component is pressed.
+   */
+  @property({ reflect: true }) pressed: boolean = false;
```

Comment:
I wonder if this could be "toggled" to match the `type` - and maybe minimize potential confusion with any css properties? (even though - I think in this case, it would be the same, ha)

Reply:
Yeah what do you think @geospatialem @jcfranco?

- type: "toggle" & "toggled"?
- type: "press" & "pressed"?
- type: "select" & "selected"?